stations/stations.go

```diff
@@ -0,0 +1,40 @@
+package stations
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+)
+
+type Station struct {
+	ID   string `json:"id"`
+	Name string `json:"name"`
+	Code string `json:"code"`
+}
+
+type CodeToID map[string]string
+
+var StationCodeToStationID CodeToID
+
+func ProcessStationMap() {
```

Comment:
Get station data from the website periodically. Get once when starting up the application then refresh the map maybe everyday or every week.